js/example.js

```diff
@@ -13,6 +13,10 @@ module.exports = function (styleguide, filePath) {
   const buildPath = styleguide.path.build()
   let data = resolver.data(buildPath, filePath)
 
+  if (Object.keys(data).length === 0) {
```

Comment:
Correct. I was hoping to provide a better error message to the developer. For example, if I let the error bubble up, it reads: `TypeError: Cannot read property 'slice' of undefined at ...` vs `Error: Example data contains zero keys at ...`.